modules/enable-monitoring-central-helm.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * configuration/monitor-acs.adoc
+:_module-type: PROCEDURE
+[id="enable-monitoring-central-helm_{context}"]
+= Monitoring Central using Helm charts
+
+[role="_abstract"]
+You can monitor Central, by changing the Central configuration options in the  the `central-services` Helm chart.
+
+.Procedure
+. Update the `values-public.yaml` configuration file with the following:
++
+[source,yaml]
+----
+exposeMonitoring: true
```

Comment:
Shouldn't this be `central.exposeMonitoring` (and `scanner.exposeMonitoring` for scanner)?